scripts/generate.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to fetch latest swagger spec.
+# Puts the updated spec at api/swagger-spec/
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if ! which mvn > /dev/null 2>&1; then
+  echo "Maven is not installed."
+  exit
+fi
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+CLIENT_ROOT=${SCRIPT_ROOT}/..
```

Comment:
Could you put double quotes around in case the variable contains spaces? There are several occurrences later as well.